samples/highcharts/chart/events-fullscreenopen-fullscreenclose/demo.details

```diff
@@ -0,0 +1,6 @@
+---
+ name: Highcharts Demo
+ authors:
+   - Michał Filipiec
```

Comment:
I suggest not to use polish letters (they don't display correctly in some cases)